File: src/main/java/gauge/language/token/SpecElementType.java
package gauge.language.token;

import com.intellij.psi.tree.IElementType;
import gauge.language.Specification;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public final class SpecElementType extends IElementType {
  public SpecElementType(@NotNull @NonNls String debugName) {
    super(debugName, Specification.INSTANCE);
  }
}

File: src/main/kotlin/gauge/language/psi/SpecElementType.kt
package gauge.language.psi

import gauge.language.SpecLanguage
import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import org.jetbrains.annotations.NotNull

class SpecElementType(@NotNull @NonNls debugName: String) :
    IElementType(debugName, SpecLanguage.INSTANCE)